content/en/bio/edwin-schmierer.md

```diff
@@ -0,0 +1,16 @@
+---
+name: "Edwin Schmierer"
+slug: "edwin-schmierer"
+profile: img/team/edwin-schmierer.png
+designation : Founder/COO
+field: The Business
+social :
+  - name : "linkedin"
+    link: 'https://www.linkedin.com/in/ESchmierer/'
```

Comment:
Oops, looks like this link is broken -- I think it should be 'https://www.linkedin.com/in/edwinschmierer/'